spdockit-insights/insights-performance.md

```diff
@@ -47,7 +53,12 @@ The server overview dashboard shows all metrics that are collected for each serv
 
 - __SQL__ category – this category is assigned to all SQL servers in SharePoint farms. Each server in Always-On cluster is included as well. These metrics show the SQL server performance.
 
-If the metric is in warning or critical state the notification above the metric name will indicate the problem. It will tell how the metric crossed the threshold value. The status of each metric is calculated based on average in the last fifteen minutes. The status of metrics for which the warning or critical thresholds are not defined will not be calculated.
+If the metric is in warning or critical state the notification above the metric name will indicate the problem. It will tell how the metric crossed the threshold value. The status of each metric is calculated based on average in the last fifteen minutes. The status of metrics for which the warning or critical thresholds are not defined will not be calculated.  
+
+Hover the mouse over any chart and in __top right corner__ you will notice the maximize button. Click it to expand the chart over the entire screen. You can toggle __Show thresholds__ button to show or hide the treshold lines (both critical and warning).  
```

Comment:
Hovering over any chart will show a maximize button in the __top right corner__. [not sure about the show verb here though]